contract/src/lib.rs

```diff
@@ -392,6 +392,12 @@ impl MpcContract {
         }
     }
 
+    #[private]
+    pub fn clean_payloads(&mut self) {
+        log!("clean_payloads");
+        self.pending_requests = LookupMap::new(b"m");
```

Comment:
were you able to test that this works? This actually doesn't do anything, since the state tree requires explicit modification, as in we need to explicitly call remove on each of the entries

Reply:
totally forgot about that, fixed